query-connector/e2e/load.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from "@playwright/test";
```

Comment:
this whole page is just a copy-paste from the existing file to help parallelize the playwright runners

Reply:
Nice! Did you notice any difference by splitting it out?

Reply:
Actually looking at the logs again it looks like we're just using the one worker, so don't think this did anything 😅

But hopefully this will help us into the future / a general positive organizational change

Reply:
haha well at least it will make it easier to go through tests!